themes/default/content/blog/pulumi-release-notes-m59/index.md

```diff
@@ -0,0 +1,68 @@
+- Pulumi Service and Pulumi.com
+  - [Get started with Pulumi, directly from the Pulumi Service]({{< relref "/blog/pulumi-release-notes-m59#get-started-with-pulumi-directly-from-the-pulumi-service" >}})
+
+<!--more-->
+
+## New and updated cloud providers
+
+### Kubernetes operator supports GitOps workflow
+
+For some Kubernetes workflows, developers want to be able to store their configuration in a Git repository but ensure that their cluster automatically stays up-to-date with the latest configuration when it's checked in. With the latest update to the Pulumi Kubernetes Operator, you can set up this type of GitOps workflow. To get started, [specify the fully-qualified branch name](https://github.com/pulumi/pulumi-kubernetes-operator/blob/80398a85958215a7c2c87e9ce30c69998f6cdba9/pkg/apis/pulumi/v1alpha1/stack_types.go#L99-L101) that you want to track in the `Stack` CustomResource (e.g., `refs/heads/main` will track the `main` branch). The operator will track the specified branch and automatically deploy new commits that are pushed to the branch.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi-kubernetes-operator/issues/50)
+
+### 12 new resources in the Azure Native provider
+
+We shipped 4 new versions of the Azure Native provider (1.16.0 through 1.19.0) that collectively added 12 new resources that you can manage with the Azure Native provider, including Fast Healthcare Interoperability Resources (FHIR) and Azure Relay resources.
+
+[See the full list](https://github.com/pulumi/pulumi-azure-native/blob/master/CHANGELOG.md#1190-2021-07-22)
+
+### Google Native: autonaming; easier management of project and region settings
+
+The preview of the Google Native provider continues to evolve. We shipped a new version of the Google Native provider ([0.5.0](https://github.com/pulumi/pulumi-google-native/releases/tag/v0.5.0)) that includes:
+
+- Full auto-naming support, just like our other providers
+- An easier way to use arguments like `machineType` that expect a fully-qualified string like `projects/${project}/zones/${zone}/machineTypes/f1-micro`: you can now use just the last value (`f1-micro`) in your program and the provider will construct the fully-qualified string
+- Automatic selection of the project, region, and zone based on your configuration, so you no longer need to pass these values to the SDK's constructor
+- An improved implementation of `BucketObject`
+
+[See the full list of updates](https://github.com/pulumi/pulumi-google-native/blob/master/CHANGELOG.md)
+
+## Pulumi CLI and core technologies
+
+In this milestone, we shipped Pulumi versions [3.7.0](https://github.com/pulumi/pulumi/releases/tag/v3.7.0), [3.7.1](https://github.com/pulumi/pulumi/releases/tag/v3.7.1), and [3.8.0](https://github.com/pulumi/pulumi/releases/tag/v3.8.0). The full list of changes in each version is available in the linked changelog; read on to learn about some of the biggest changes.
```

Comment:
Perhaps worth expanding on here or in their own section:
* https://github.com/pulumi/pulumi/pull/7483
* Resource Methods

Reply:
Methods will get its own blog post in M60, as well as a shoutout here. I'll add the `Output[T]` one though.